mandatory/3-playing-computer.js

```diff
@@ -6,13 +6,13 @@
 
   Answer the following questions:
 
-  1. This program throws an error. Why? (If you can't find it, try executing it).
-  2. Remove the line that throws the error.
-  3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  1. This program throws an error. Why?Because b is  undefined.(If you can't find it, try executing it).
+  2. Remove the line that throws the error.I did.
+  3. What is printed to the console? 
+  4. How many times is "f1" called? 2 times
+  5. How many times is "f2" called? 3 time but I am not sure and I have question.
+  6. What value does the "a" parameter take in the first "f1" call?6
```

Comment:
it would be 1. a in the f1 is equal to the i outer variable. In the first iteration i = 0, but f2 does not get called. But in the second iteration f2 = 1 and then it gets called.